spec/controllers/api/v2/api_works_spec.rb

```diff
@@ -11,12 +11,12 @@
     before :all do
       mock_external
     end
-    
+
     before :each do
       Rails.cache.clear
```

Comment:
Should we remove this step? We do that at:

https://github.com/otwcode/otwarchive/blob/c8fe1392b0493dfd5730be590d3b60715414791e/spec/spec_helper.rb#L96